src/main/java/az/gdg/msalarm/client/MsAlarmClient.java

```diff
@@ -0,0 +1,10 @@
+package az.gdg.msalarm.client;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@FeignClient(value = "ms-alarm-client", url = "https://gdg-ms-alarm.herokuapp.com/alarm")
+public interface MsAlarmClient {
+    @GetMapping
+    void invokeMsAlarm();
```

Comment:
invokeAlarm is enough, do not use Ms wording whenever you can, use it only where needed.